lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_bump_version_hotfix.rb

```diff
@@ -10,17 +10,17 @@ def self.run(params)
         app = params[:app]
 
         current_version = Fastlane::Helper::Android::VersionHelper.get_release_version(product_name: app)
-        current_version_alpha = Fastlane::Helper::Android::VersionHelper.get_alpha_version(app)
-        new_version = Fastlane::Helper::Android::VersionHelper.calc_next_hotfix_version(params[:version_name], params[:version_code])
-        new_short_version = new_version_name
-        new_release_branch = "release/#{new_short_version}"
+        new_version = Fastlane::Helper::Android::VersionHelper.calc_next_hotfix_version(params[:version_name], params[:version_code]) # NOTE: this just puts the name/code values in a tuple, unchanged (no actual calc/bumping)
+        new_release_branch = "release/#{params[:version_name]}"
 
-        UI.message("Current version[#{app}]: #{current_version[Fastlane::Helper::Android::VersionHelper::VERSION_NAME]}(#{current_version[Fastlane::Helper::Android::VersionHelper::VERSION_CODE]})")
-        UI.message("New hotfix version[#{app}]: #{new_version[Fastlane::Helper::Android::VersionHelper::VERSION_NAME]}(#{new_version[Fastlane::Helper::Android::VersionHelper::VERSION_CODE]})")
+        name_key = Fastlane::Helper::Android::VersionHelper::VERSION_NAME
+        code_key = Fastlane::Helper::Android::VersionHelper::VERSION_CODE
+        UI.message("Current version [#{app}]: #{current_version[name_key]} (#{current_version[code_key]})")
+        UI.message("New hotfix version[#{app}]: #{new_version[name_key]} (#{new_version[code_key]})")
```

Comment:
We really need to implement the refactor planned in #203 at some point that would make all that way nicer to read… but that's for a separate time 😞